CollectionView/ViewController.swift

```diff
@@ -87,3 +82,38 @@ extension ViewController: UICollectionViewDelegateFlowLayout {
         return UIEdgeInsets(top: 35, left: 20, bottom: 0, right: 20)
     }
 }
+extension UIImage {
```

Comment:
We usually have extensions that can be used in more than one place in exclusive files. Also, these files are usually named `[class_name]+[ability_added].swift`. In this case, this file should be named `UIImage+Resize.swift`.